test/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.13.4)
+cmake_minimum_required(VERSION 3.25)
 project(hello)
 
 set(hello_SOURCES hello.c)
```

Comment:
@mstorsjo Is the `WIN32` condition necessary? Is this cmakelist supposed to target platforms other than Windows?

Reply:
I'd like the test to remain compileable for any platform, yeah. It's not something that we'd do much, admittedly, but I'd like the cmakelist to remain like a real one would be for a proper cross platform project.